### PR TITLE
UI check to avoid peering a same managed cluster with more than one cluster

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Issue: https://issues.redhat.com/browse/RHSTOR-6634

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
Screenshot 2024-11-13 at 8 12 22 PM

File: packages/mco/components/create-dr-policy/selected-cluster-validator.tsx [hunk 1]
```diff
@@ -57,9 +57,26 @@ const checkClientToODFPeering = (clusters: ManagedClusterInfoType[]): boolean =>
   !!clusters[0]?.odfInfo?.storageClusterInfo?.clientInfo !==
   !!clusters[1]?.odfInfo?.storageClusterInfo?.clientInfo;
 
+const verifyMirrorPeerExistance = (
```
Comment (alfonsomthd) [inline packages/mco/components/create-dr-policy/selected-cluster-validator.tsx:60]:
```suggestion
const verifyMirrorPeerExistence = (
```

**Pull request merged**